website/mails/mails.py

```diff
@@ -107,7 +112,8 @@ def send_mail(to_addr, mail, mimetype='plain', from_addr=None, mailer=None,
         login=login,
         username=username,
         password=password,
-        mail_server=mail_server
+        mail_server=mail_server,
```

Comment:
Looks like the `mail_server` key is used only in [`scripts/admin_permission_email.py`](https://github.com/CenterForOpenScience/osf.io/blob/25a97810cc557d8dc1792279a29d5b5c59656a68/scripts/admin_permission_email.py) (cf. https://github.com/CenterForOpenScience/osf.io/pull/4740#discussion_r50793219). If we can get rid of that script then it seems that we can get rid of this kwarg as well.